.github/workflows/coverage.yml

```diff
@@ -27,46 +27,33 @@ jobs:
       - name: Checkout repository,
         uses: actions/checkout@v3
 
+      - uses: dtolnay/rust-toolchain@nightly
+      - uses: dtolnay/rust-toolchain@stable
```

Comment:
No idea, but could it be a problem to use two versions at the same time?

Reply:
For this workflow I need both versions, stable for compiling `sqlx-cli` and nightly for computing the test coverage.